gramps/gen/filters/rules/family/_fatherhasnameof.py

```diff
@@ -47,5 +55,13 @@ class FatherHasNameOf(HasNameOf):
     name = _("Families with father with the <name>")
     description = _("Matches families whose father has a specified " "(partial) name")
     category = _("Father filters")
-    base_class = HasNameOf
-    apply = father_base
+
+    def apply_to_one(self, db: Database, family: Family) -> bool:  # type: ignore[override]
+        father_handle = family.father_handle
+        if father_handle:
+            father = db.get_person_from_handle(father_handle)
+            if father:
+                return super().apply_to_one(db, father)
+            else:
+                return False
```

Comment:
```suggestion
            return False
```
Unnecessary else.

Reply:
I think it is more explicit, and "Explicit is better than implicit."